packages/obonode/obojobo-chunks-iframe/editor-component.js

```diff
@@ -71,13 +71,21 @@ class IFrame extends React.Component {
 	}
 
 	getTitle(src, title) {
+		const charLimit = 50
+
 		if (src === null) {
 			return 'IFrame missing src attribute'
 		} else if (title) {
 			return title
```

Comment:
This also needs to be limited - This code is ran if the user supplies their own title